driver/driver.go

```diff
@@ -236,7 +236,7 @@ func logGRPC(method string, request, reply interface{}, err error) {
 	}
 
 	msg, _ := json.Marshal(logMessage)
-	fmt.Printf("gRPCCall: %s\n", msg)
+	klog.V(3).Infof("gRPCCall: %s\n", msg)
```

Comment:
There is code in Kubernetes which parses this output: https://github.com/kubernetes/kubernetes/blob/ec560b9737537be8c688776461bc700e8ddedb9d/test/e2e/storage/csi_mock_volume.go#L1354-L1360

I suspect using klog here will break that.

I'm leaning towards doing this (and only this) output with `fmt.Printf`.

Reply:
Yeah, that makes sense. Just wondering if there is anyway we can add timestamps to these logs without breaking the e2e tests? The context is that it might be good for us to understand when it receives the gRPCCall for triage some issues.

Reply:
The alternative is to make theE2E tests more flexible such they support such lines with and without the standard klog prefix, then update the mock driver.

Reply:
It seems the E2E test is using a static version of mock CSI driver which is v3.1.0. So unless we update the driver version there, we should not break the test. 

Should the right order be: 
1. Update the mock CSI driver to use log.
2. Update the e2e test to use latest driver version tag and also apply changes to the test cases?

Reply:
I prefer to update the E2E test in advance, because then at some point in the future, the mock driver image can be updated without having to remember that the change is still needed.

Reply:
https://github.com/kubernetes/kubernetes/pull/93950
PR is out.